### DISCUSSION TITLE
1.102 Breaking Change

### DISCUSSION THREAD

**Discussion by script-js** [immich-app/immich#8984] [Q&A] [answered]
I saw that there was a breaking change listed in the 1.102 update that said it would only affect me if I always updated to the latest docker compose file. Is there a way to check if I'm always updating? I know this is probably a stupid question but I'm not very familiar with docker or most server/database software.

**Comment by mmomjian** [comment, marked as answer]
Unless you personally edit the contents of your docker-compose.yml file it does not affect you.

**Reply by script-js** [reply]
Thank you! This helped a lot!
